src/core/lib/security/credentials/external/external_account_credentials.h

```diff
@@ -51,6 +51,7 @@ class ExternalAccountCredentials
     std::string audience;
     std::string subject_token_type;
     std::string service_account_impersonation_url;
+    Json service_account_impersonation;
```

Comment:
Any specific reason to store this as a Json object? It'd be more efficient to have parsed content here so that subsequent calls (ImpersonateServiceAccount) don't have to parse the Json again.

Reply:
Modified this to store parsed content here. Thank you.